cli/src/commands/synchronousOnboarding.ts

```diff
@@ -0,0 +1,60 @@
+import { from, interval, Observable } from "rxjs";
+import { withDevice } from "@ledgerhq/live-common/lib/hw/deviceAccess";
+import getOnboardingStatus from "@ledgerhq/live-common/lib/hw/getOnboardingStatus";
+import { concatMap, tap } from "rxjs/operators";
+
+export default {
+  description:
+    "track the onboarding status of your nano",
+  args: [
+    {
+      name: "refresh",
+      alias: "r",
+      desc: "refresh rate in milliseconds",
+    },
+  ],
+  job: ({
+    device,
+  }: Partial<{
+    device: string;
+  }>) => interval(1000).pipe(
+      concatMap(() => withDevice(device || "")((t) => from(getOnboardingStatus(t)))),
```

Comment:
If you want a helper to parse the falgs it's fine, but you shouldn't be exchanging apdus twice to get the same data.
You are essentially doubling the work for no reason